docs/design-proposals/multi-tenancy/multi-tenancy-network-mizar-integration.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Arktos Multi-Tenancy Network Based on Mizar
+authors:
+- "@sindica"
+---
+
+# Arktos Multi-Tenancy Network Based on Mizar
+
+## Table of Content
+1. [Introduction](#intro)
+2. [Goals](#goal)
+3. [Proposal](#proposal)<br>
+   3.1. [Related Mizar Specs](#mizar-spec)<br>
+   3.2. [Related Arktos Specs](#arktos-spec)<br>
+   3.3. [Arktos Pod Creation Workflow](#pod-creation-wf)<br>
+   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3.3.1 [Workflow for default VPC and subnet in tenant](#pod-creation-default-wf)<br>
+   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3.3.2 [Workflow for secondary VPC and subnet in tenant](#pod-creation-second-vpc-wf)<br>
+   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3.3.3 [Workflow for secondary subnet in VPC](#pod-creation-second-subnet-wf)<br>
+
+## 1. Introduction <a name="intro"></a>
+
+This document is a part of the multi-tenancy network design document that focuses on how to integrated [Mizar network solution](https://github.com/CentaurusInfra/mizar) into Arktos.
+
+The generic network design is in [Multi-Tenancy Network](multi-tenancy-network.md).
+
+## 2. Goals <a name="goal"></a>
+
+* Primary: support multi-tenancy communication based on existing Mizar network implementation. Multi-tenancy communication 
+includes connectivity amongs pods in the same tenant, as well as network isolation across different tenants.
+* Primary: upon tenant creation, automatically create Mizar VPC and subnet for future communication within this tenant.
+* Primary: upon tenant pod creation, automatically attach pod to Mizar VPC/subnet created for this tenant.
+* Primary: Mizar and Arktos decouple, i.e. Mizar manage VPCs and subnets in VPC only, it should not be aware of the ownership
+of different pods and other objects.
+* Secondary: support multiple subnets in same VPC for a single tenant. (Post 2022-01-30)
+* Secondary: support multiple VPC/subnets in same tenant. (Post 2022-01-30)
+
+## 3. Proposal <a name="proposal"></a>
+
+Inject Mizar adapter into Arktos, completely isolated from Mizar management plane and relatively isolated from Arktos network
+management components.
+
+### 3.1. Related Mizar Specs <a name="mizar-spec"></a>
+
+* VPC Spec
+```
+apiVersion: "mizar.com/v1"
```

Comment:
is mizar.com approved by the cloud lab?

Reply:
> is mizar.com approved by the cloud lab? or this should be better align with the arktos.futurewei.com below as mizar.futurewei.com.

https://mizar.readthedocs.io/en/latest/user/getting_started/

This should be a question to mizar team.